edgelet/edgelet-http/Cargo.toml

```diff
@@ -8,21 +8,19 @@ edition = "2021"
 [dependencies]
 anyhow = "1"
 chrono = "0.4"
-futures-util = "0.3"
 http = "0.2"
 libc = "0.2"
 log = "0.4"
 percent-encoding = "2"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+tokio = { version = "1", features = ["macros", "parking_lot", "sync"] }
 
 edgelet-core = { path = "../edgelet-core" }
 edgelet-settings = { path = "../edgelet-settings" }
 
 http-common = { git = "https://github.com/Azure/iot-identity-service", branch = "main" }
 
 [dev-dependencies]
-tokio = { version = "1", features = ["macros"] }
```

Comment:
The tests use `#[tokio::test]`. The crate compiles for now since `edgelet-utils` uses the `macros` and `rt` features[^1], but `macros` and `rt` should be specified here to prevent future issues.

[^1]: https://docs.rs/tokio/latest/tokio/attr.test.html

Reply:
Fixed